src/components/Sum.jsx

```diff
@@ -1 +1,21 @@
-// export function Sum() {}
+import React from 'react';
+
+function Sum({ a = 0, b = 0 }) {
```

Comment:
export this component directly to make tests work correctly. And it's better to use one way to declare components in the project (function expression or function declaration). It's different from the App component now
```suggestion
export const Sum = ({ a = 0, b = 0 }) => {
```